src/benchmark.cc

```diff
@@ -575,6 +621,13 @@ void BenchmarkImp::Repetitions(int n) {
   repetitions_ = n;
 }
 
+void BenchmarkImp::ReportAggregatesOnly(bool value) {
+  if (report_options_ == RO_Unspecified)
+      report_options_ = RO_Default;
+  if (value) report_options_ &= ~RO_ReportAggregatesOnly;
```

Comment:
Done. The operator overloads are removed and  the report options are no longer treated as a bit mask.